README.md

```diff
@@ -113,37 +138,27 @@ Name it after what users will see, not what it represent in your application.
 <ae-category>Category 1</ae-category>
 ```
 
-> Some developer might want to use the same visual element to display `tags` or `online statuses`. So she would end up writing `<ae-category>Label 1</ae-category>` or `<ae-category>Offline</ae-category>`🤦🏻‍♀️
+> Some developer might want to use the same visual element to display `tags` or `online statuses`. So she would end
+up writing `<ae-category>Label 1</ae-category>` or `<ae-category>Offline</ae-category>`🤦🏻‍♀️
 
 ### Component file name and location
-Put the files in a camelcased directory inside `/src/components` like `/src/components/aeComponentName/`.
-Create a .vue file with the same name (ie. `aeComponentName.vue`) that includes template, script and style. Script and style may also be imported from separate files like `aeComponentName.js` and `aeComponentName.(s)css`
+Put the files in a `kebab-case` directory inside `src/components` like `src/components/ae-component-name/`.
+Create a .vue file with the same name (ie. `ae-component-name.vue`) that includes template, script and style. 
+Please bundle the `scss` and `javascript` code within the component file a single vue component. Tests can go
+in an external file named `ae-components.test.js`.
 
 ```
 aepp-components
 └── src
     └── components
-        └── aeComponentName
-            ├── aeComponentName.js
-            ├── aeComponentName.md
-            ├── aeComponentName.scss
-            ├── aeComponentName.vue
-            └── index.js
+        └── ae-component-name
+            ├── ae-component-name.md
+            ├── ae-component-name.test.js
+            └── ae-component-name.vue
```

Comment:
I still don't agree with kebab case in component naming. I thought that we are using it to distinguish new and old components

Reply:
We have a talk with Emil in September and as I understand we decided to revert kebab-case and use CamelCase as it was before.

Reply:
> We have a talk with Emil in September and as I understand we decided to revert kebab-case and use CamelCase as it was before.

I remember differently, we agreed to 90% of your requests, but the naming of the components we agreed to keep it kebab-case.

Reply:
This looks fine to me. I don't see any big cons in using kebab-case naming for file names.